### PR TITLE
Create cohenSutherlandAlgorithm.cpp

### PR DESCRIPTION
**Pull request by piyush26c**
Add implementation of CohenSutherland Algorithm in C++.

### CODE REVIEW

File: code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,156 @@
+#include <iostream>
```
Comment (arnavb) [inline code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: Please follow our C++ style guide located [here](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c%2B%2B).

Comment (piyush26c) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: I have updated changes which you have suggested please, review it.

Comment (piyush26c) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: I have updated changes which you have suggested please, review it.

File: code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp [hunk 2]
```diff
@@ -127,7 +127,7 @@ void CohenSutherLandAlgo::cohenSutherland()
                 y2_ = y;
                 code2 = generateCode(x2_, y2_);
             }
-        }
+        }
```
Comment (arnavb) [inline code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: What is the purpose of this?

Comment (arnavb) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: No, I was asking why you added an extra space at the end of line. Trailing whitespace adds nothing.

Comment (piyush26c) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: Ohh, sorry for wrong interpretation, I will remove it.

Comment (piyush26c) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: It belongs to indentation...

Comment (arnavb) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: How does the space after the brace belong to indentation? The spaces before the brace definitely to, but the trailing whitespace serves no purpose.

```suggestion
        }
```

Comment (piyush26c) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: Yeah it is true I had wrong interpretation here... I will remove it surely.

Comment (piyush26c) [reply code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp]: I had made the changes as per your suggestion please review it,

**Comment by arnavb** [comment]
djldk 2

**Pull request merged**